README.md

```diff
@@ -399,6 +400,39 @@ or `EntityManagerFactory`, you can use this code to access the underlying `Sessi
 ```java
 SessionFactory sessionFactory = entityManagerFactory.unwrap(SessionFactory.class);
 ```
+### Jetty
+
+There is a collector for recording various Jetty server metrics. You can do it by  registering the collector like this:
+
+```java
+// Configure StaticticsHandler.
+StatisticsHandler stats = new StatisticsHandler();
+stats.setHandler(server.getHandler());
+server.setHandler(stats);
+// Register collector.
+new JettyStatisticsCollector(stats).register();
+
+```
+
+Also, you can collect `QueuedThreadPool` metrics. If there is a single `QueuedThreadPool`
+to keep track of, following lines work: 
+
+```java
+new QueuedThreadPoolStatisticsCollector(queuedThreadPool, "myapp").register();
+```
+
+If you want to collect multiple `QueuedThreadPool` metrics, also you can achieve it like this:
+
+```java
+new QueuedThreadPoolStatisticsCollector()
+    .add(queuedThreadPool1, "myapp1")
+    .add(queuedThreadPool2, "myapp2")
+    .register();
+```
+
+Note: If you are using 9.3.x or more recent Jetty version which requires Java 8,
```

Comment:
This isn't quite right. If you want certain features you'd need both. As we don't currently state which module you need for what (and a github search will tell you), I'd say leave it out.